<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestioneMateriale</class>
 <widget class="QDialog" name="gestioneMateriale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>383</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>383</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>610</width>
    <height>383</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestione Materiale</string>
  </property>
  <widget class="QComboBox" name="comboBoxCercaCategoria">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnElencoEsci">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Esci</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCerca">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>20</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerca</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMostraCategoria">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Mostra elenco per categoria:</string>
   </property>
  </widget>
  <widget class="QTableView" name="tblMateriale">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>471</width>
     <height>141</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btnEsci">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>320</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Esci</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPulisci">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>60</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Pulisci</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="grpNuovoMateriale">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>471</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Nuovo Materiale</string>
   </property>
   <widget class="QLabel" name="lblCategoria">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Categoria</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTipologia">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipologia</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDimensione">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Dimensione</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxCategoria">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BUSTE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SECCHIO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MASTELLO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CALENDARIO</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxTipologia">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSalva">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Salva</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxDimensione">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnElimina">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>100</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Elimina</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
